macros/utils/graph/get_result_node.sql

```diff
@@ -0,0 +1,14 @@
+{% macro get_result_node(identifier, package_name='elementary') %}
+  {% for result in results %}
+    {% if result.node.identifier == identifier %}
```

Comment:
Are you sure all node types have an identifier? in `get_model_graph_node` we checked identifier / name / alias to be on the safe side

Reply:
<img width="773" alt="image" src="https://user-images.githubusercontent.com/30181361/197387577-0a69f844-1d3b-492a-a6a5-babc0ffdb195.png">

All parsed nodes have an `identifier` property.